scripts/kani-regression.sh

```diff
@@ -97,6 +97,11 @@ time "$SCRIPT_DIR"/codegen-firecracker.sh
 # Test run 'cargo kani assess scan'
 "$SCRIPT_DIR"/assess-scan-regression.sh
 
+# Test for --manifest-path which we cannot do through compiletest.
+# It should just successfully find the project and specified proof harness. (Then clean up.)
+cargo kani --manifest-path "$KANI_DIR/tests/cargo-kani/cargo-features-flag/Cargo.toml" --harness trivial_success
```

Comment:
nit: define the harness path as a variable to avoid any issues due to typos

Reply:
done